tekton/cd/results/overlays/dogfooding/service.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    cloud.google.com/backend-config: '{"default": "tekton-results-api"}'
+    cloud.google.com/neg: '{"ingress": true}'
+    cloud.google.com/app-protocols: '{"grpc":"HTTP2"}'
+  name: tekton-results-api-service
```

Comment:
is it possible to include some comments or docs explaining how this service relates to [this service](https://github.com/tektoncd/results/blob/9c6dd2231d9e05bf55bb371e966e3e98295f2149/config/api.yaml#L72-L88)? (i assume that this config is augmenting the service config from the results repo? im not quite understanding how these fit together 🙏 )

Reply:
Added a small readme to show how to preview/diff the changes, also changed the base to a remote resource to make it clearer where these resources are coming from.

Hopefully this is enough? 🤞  Let me know!

Reply:
ah okay, so this is a 'kustomization' on top of a published release.yaml? i think ive got it! thanks for adding the readme